templates/zerver/apps.html

```diff
@@ -86,10 +86,17 @@ <h3 class="center">{% trans %}Ok, I take it back, I'm sorry, please don't go.{%
                             <div class="wrapper">
                                 <h2>MacOS</h2>
                                 <div class="instructions">
-                                    <p class="center">You love your Mac. And you love Zulip. So what could be better than a Zulip app for Mac? Enjoy notifications for messages and PMs in your dock whether you're in Sublime, emacs, or Photoshop.</p>
+                                    <p class="center">
+                                        {% trans %}You love your Mac. And you
+                                        love Zulip. So what could be better
+                                        than a Zulip app for Mac? Enjoy
+                                        notifications for messages and PMs in
+                                        your dock whether you're in Sublime,
+                                        emacs, or Photoshop.{% endtrans %}
+                                    </p>
```

Comment:
How about we leave these as single long lines? It looks less nice that way in an editor, but from my small experience trying translating things with Transifex, I think the newlines end up being a pain to work with.